samples/Azure.Management.Storage/Generated/RestOperations/BlobContainersRestOperations.cs

```diff
@@ -29,17 +28,17 @@ internal partial class BlobContainersRestOperations
         /// <summary> Initializes a new instance of BlobContainersRestOperations. </summary>
         /// <param name="clientDiagnostics"> The handler for diagnostic messaging in the client. </param>
         /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
-        /// <param name="options"> The client options used to construct the current client. </param>
+        /// <param name="applicationId"> The application id to use for user agent. </param>
         /// <param name="endpoint"> server parameter. </param>
         /// <param name="apiVersion"> Api Version. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="apiVersion"/> is null. </exception>
-        public BlobContainersRestOperations(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, ArmClientOptions options, Uri endpoint = null, string apiVersion = default)
+        public BlobContainersRestOperations(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string applicationId, Uri endpoint = null, string apiVersion = default)
```

Comment:
I think you can get rid of this clientDiagnostics parameter by using the `new RequestFailedException(response)` to create exceptions.

Reply:
I will create a follow up item for this since its all internal

Reply:
https://github.com/Azure/autorest.csharp/issues/1897